src/utils/localStorage.ts

```diff
@@ -1,2 +1,20 @@
-export const accessTokenKey: (venueId: string) => string = (venueId) =>
-  `token-${venueId}`;
+import firebase from "firebase/app";
+
+type CheckAccessTypes = {
+  venueId: string;
+  password?: string;
+  email?: string;
+  code?: string;
+  token?: string;
+};
+
+export const accessTokenKey = (venueId: string) => `token-${venueId}`;
+
+export const checkAccess = async (data: CheckAccessTypes) =>
+  await firebase.functions().httpsCallable("access-checkAccess")(data);
```

Comment:
`checkAccess` doesn't relate to `localStorage` at all, it should be in a file under `api/` somewhere.

Reply:
agreed on that ^. Let's keep our firebase\external calls in one place